plugin/build_baron.go

```diff
@@ -208,20 +208,12 @@ func BbGetProject(settings *evergreen.Settings, projectId string, version string
 		return evergreen.BuildBaronSettings{}, false
 	}
 	if flags.PluginAdminPageDisabled {
-		if version == "" {
-			lastGoodVersion, err := model.FindVersionByLastKnownGoodConfig(projectId, -1)
-			if err == nil && lastGoodVersion != nil {
-				version = lastGoodVersion.Id
-			}
+		projectConfigs, err := model.FindProjectConfigToMerge(projectId, version)
+		if err != nil {
+			return evergreen.BuildBaronSettings{}, false
 		}
-		if version != "" {
-			parserProject, err := model.ParserProjectFindOneById(version)
-			if err != nil {
-				return evergreen.BuildBaronSettings{}, false
-			}
-			if parserProject != nil && parserProject.BuildBaronSettings != nil {
-				return *parserProject.BuildBaronSettings, true
-			}
+		if projectConfigs != nil && projectConfigs.BuildBaronSettings != nil {
```

Comment:
This should come after ProjectRef since that now takes precedence, but idk if you have a different ticket to handle that.

Reply:
Actually since findMergedProjectRef already takes care of this we can remove the entire block looking up project config from line 211 - 217. Removing.